### PR TITLE
Removed condescending language from Command Reference

### LINKED ISSUE #1394: docs: remove condescending/scary language
> Continuation of #608

Here's a good article about this: https://dev.to/meeshkan/how-to-remove-condescending-language-from-documentation-4a5p#what

TL;DR: Presenting something as "extremely simple" (with the intention of selling a benefit) may result in the reader feeling intimidated, esp. if they don't find the topic simple.

Examples

```
simply
easy
basically
clearly
everyone knows
just
obviously
of course

As you/we can see, ...
It's well known
```

Some of these have legitimate uses like "just" as a quantifier e.g. _just one thing_ vs. as a qualifier e.g. _you can just xyz_.

To fix: _simply_ removing it works in most cases. If the intention was to emphasize something, be more descriptive e.g. _now this can be easily done_ -> _this feature aims to makes the process easier_.

### PR DESCRIPTION
**Pull request by mformihir**
Address #1394 (partially)

- Reviewed and made changes to [Command Reference](https://dvc.org/doc/command-reference).
- I fixed a couple of typos and minimized simply removing all the occurrences of 'just'.

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
@mformihir thank! I'll review this now. Please don't use the special word "Fix" in the description since it's only partial. See https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

File: content/docs/command-reference/checkout.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ baseline-experiment     <- First simple version of the model
 bigrams-experiment      <- Uses bigrams to improve the model
 ```
 
-We can now just run `dvc checkout` that will update the most recent `model.pkl`,
+We can now run `dvc checkout` that will update the most recent `model.pkl`,
```
Comment (jorgeorpinel) [inline content/docs/command-reference/checkout.md:161]:
```suggestion
We can now run `dvc checkout` to update the most recent `model.pkl`,
```
The paragraph will need formatting after this change though.

> Same as in https://github.com/iterative/dvc.org/pull/1396/files#diff-d159337499c6716e1d89e1c12b5a8d09

File: content/docs/command-reference/commit.md [hunk 2]
```diff
@@ -96,8 +96,8 @@ reproducibility in those cases.
 
 ## Examples
 
-Let's employ a simple <abbr>workspace</abbr> with some data, code, ML models,
-pipeline stages, such as the <abbr>DVC project</abbr> created for the
+Let's employ a <abbr>workspace</abbr> with some data, code, ML models, pipeline
+stages, such as the <abbr>DVC project</abbr> created for the
```
Comment (jorgeorpinel) [inline content/docs/command-reference/commit.md:100]: I'm not sure this is condescending but no strong opinion, we can change it. However, I believe this same paragraph exists in other refs. such as in checkout so let's apply the change consistently everywhere please.

Comment (jorgeorpinel) [reply content/docs/command-reference/commit.md:100]: @mformihir what about the use of "just" in this doc? See https://github.com/iterative/dvc.org/pull/1396/files#diff-aac9973cd919dc8d6735e0de82ab7f62L47-R47 Do you think it's condescending?

Comment (jorgeorpinel) [reply content/docs/command-reference/commit.md:100]: > no strong opinion, we can change it

On this, also. I'm not sure the phrase "Let's employ a workspace with some data..." reads right, without the adjective. Let's change the word "simple" for something more specific such as "new" or "fresh" please.

File: content/docs/command-reference/fetch.md [hunk 3]
```diff
@@ -120,8 +120,8 @@ or `-T` options are used).
 
 ## Examples
 
-Let's employ a simple <abbr>workspace</abbr> with some data, code, ML models,
-pipeline stages, such as the <abbr>DVC project</abbr> created for the
+Let's employ a <abbr>workspace</abbr> with some data, code, ML models, pipeline
+stages, such as the <abbr>DVC project</abbr> created for the
```
Comment (jorgeorpinel) [inline content/docs/command-reference/fetch.md:124]: Same here: new workspace; fresh workspace; clean workspace — something like that

File: content/docs/command-reference/fetch.md [hunk 4]
```diff
@@ -256,8 +256,8 @@ $ dvc status -c
     deleted:            data/data.xml
 ```
 
-One could do a simple `dvc fetch` to get all the data, but what if you only want
-to retrieve the data up to our third stage, `train.dvc`? We can use the
+One could do a `dvc fetch` to get all the data, but what if you only want to
```
Comment (jorgeorpinel) [inline content/docs/command-reference/fetch.md:259]:
```suggestion
We could use `dvc fetch` to get all the data, but what if you only want to
```

File: content/docs/command-reference/import-url.md [hunk 5]
```diff
@@ -31,7 +31,7 @@ external data source changes. Example scenarios:
 - A shared dataset on a remote storage that is managed and updated outside DVC.
 
 > Note that `dvc get-url` corresponds to the first step this command performs
-> (just download the file or directory).
+> (just downloads the file or directory).
```
Comment (jorgeorpinel) [inline content/docs/command-reference/import-url.md:34]: Good. But what about the other change from https://github.com/iterative/dvc.org/pull/1396/files#diff-06ee43874ed57c5e9a7512621fba5ed1L81-R81 ?

`` `remote://myremote/path/to/file` notation just means`` -> ``The `remote://myremote/path/to/file` notation means`` please

**Review by jorgeorpinel — DISMISSED** [review]
Thanks for the small typo and grammar corrections 🙂 

This PR is big... I only reviewed your first 6 changed files for now. Please try to apply my comments to all the other files I didn't review yet, where/if applicable.

Also, please copy the changes that you think are valid from https://github.com/iterative/dvc.org/pull/1396/files.

**Comment by jorgeorpinel** [comment]
Hi @mformihir, will you be continuing this contribution? Thanks

**Comment by shcheklein** [comment]
Closing this as stale.

**Pull request closed without merging**